site/docs/reference/Connectors/materialization-connectors/amazon-s3-csv.md

```diff
@@ -0,0 +1,96 @@
+
+### Properties
+
+#### Endpoint
+
+| Property                  | Title                 | Description                                                                                                                                   | Type    | Required/Default |
+|---------------------------|-----------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|---------|------------------|
+| **`/bucket`**             | Bucket                | Bucket to store materialized objects.                                                                                                         | string  | Required         |
+| **`/awsAccessKeyId`**     | AWS Access Key ID     | Access Key ID for writing data to the bucket.                                                                                                 | string  | Required         |
+| **`/awsSecretAccessKey`** | AWS Secret Access key | Secret Access Key for writing data to the bucket.                                                                                             | string  | Required         |
+| **`/region`**             | Region                | Region of the bucket to write to.                                                                                                             | string  | Required         |
+| **`/uploadInterval`**     | Upload Interval       | Frequency at which files will be uploaded.                                                                                                    | string  | 5m               |
+| `/prefix`                 | Prefix                | Optional prefix that will be used to store objects.                                                                                           | string  |                  |
+| `/fileSizeLimit`          | File Size Limit       | Approximate maximum size of materialized files in bytes. Defaults to 10737418240 (10 GiB) if blank.                                           | integer |                  |
+| `/endpoint`               | Custom S3 Endpoint    | The S3 endpoint URI to connect to. Use if you're materializing to a compatible API that isn't provided by AWS. Should normally be left blank. | string  |                  |
+| `/csvConfig/delimiter`    | Delimiter             | Character to separate columns within a row. Defaults to a comma if blank. Must be a single character with a byte length of 1.                 | integer |                  |
+| `/csvConfig/nullString`   | Null String           | String to use to represent NULL values. Defaults to an empty string if blank.                                                                 | integer |                  |
+| `/csvConfig/skipHeaders`  | Skip Headers          | Do not write headers to files.                                                                                                                | integer |                  |
```

Comment:
Could this parameter instead be something like `skipRows`? Often csvs have multi-row headers, not sure if we support that

Reply:
We are writing the CSV files here, and we either write them with a single row header, or don't write a header at all if `skipHeaders` is true.

Reply:
Uhh my bad, for some reason I was totally thinking this is a capture 🤦